scene/resources/particle_process_material.cpp

```diff
@@ -719,7 +719,7 @@ void ParticleProcessMaterial::_update_shader() {
 			code += "	local_pos -= velocity_pivot;\n";
 			code += "	local_pos.z = 0.;\n";
 			code += "	mat3 z_rotation_mat = mat3(\n";
-			code += "		vec3(cos(orbit_velocities.z),-sin(orbit_velocities.z),0.0),\n";
+			code += "		vec3(cos(orbit_velocities.z),sin(orbit_velocities.z),0.0),\n";
```

Comment:
Yes, this is correct, I approve. But I won't click the approve button on the PR, because other changes were added and I am not qualified to approve those.

Reply:
You can approve and say in the comment which part of the PR you're approving, same as you would do if you're only doing a code review without functional testing, or vice versa.

We read those ;)